src/tracks/playabletrack/wavetrack/ui/WaveTrackControls.cpp

```diff
@@ -261,7 +261,7 @@ void FormatMenuTable::OnFormatChange(wxCommandEvent & event)
 
    sampleCount totalSamples{ 0 };
    for (const auto& channel : TrackList::Channels(pTrack))
-      totalSamples += channel->GetNumSamples();
+      totalSamples += channel->GetSequenceSamplesCount();
```

Comment:
The unusual case where a call to one of the *Sequence* functions is needed not *Play*.  Comment it.